ingest/src/main/java/com/connexta/ingest/config/ApplicationConfiguration.java

```diff
@@ -18,8 +20,27 @@
 @EnableAutoConfiguration
 @Profile("default")
 public class ApplicationConfiguration {
+
```

Comment:
In case I forget to ask, could someone explain how @Profile("profileName") works?